apps/web/components/ui/ProductCard/ProductCard.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <div
-    class="border border-neutral-200 rounded-md hover:shadow-lg flex flex-col flex-auto flex-shrink-0"
-    data-testid="product-card"
-  >
+  <div class="border border-neutral-200 rounded-md hover:shadow-lg flex flex-col flex-1" data-testid="product-card">
```

Comment:
the following changes will fix the issue:
[https://github.com/plentymarkets/plentyshop-pwa/pull/407](https://github.com/plentymarkets/plentyshop-pwa/pull/407) 

```addModernImageExtension``` removed, as extension trigger has already being handled on parent component